src/storage/txn/action/mod.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! This file contains the "actions" we perform on a [`MvccTxn`] and related tests
+//! "Actions" here means a group of more basic operations,
+//! eg. [`MvccTxn::load_lock`], [`MvccTxn::put_write`], which are methods on [`MvccTxn`],
+//! for archiving a certain target
+
+pub mod commit;
```

Comment:
either we should just keep the commit function in this file, or we should re-export commit here so that users don't need to write `action::commit::commit`. Also I think `actions` is better than `action` for the module name.